docker-sync.yml.dist: fix relative path to postgres-data folder

DavidKuna opened the pull request:
| Q             | A
| ------------- | ---
|Description, reason for the PR| Docker sync fails due to non-existent directory path: `Starting native_osx for sync shopsys-framework-postgres-data-sync``/Library/Ruby/Gems/2.3.0/gems/docker-sync-0.5.7/lib/docker-sync/sync_strategy/native_osx.rb:41:in 'realpath': No such file or directory @ realpath_rec - /Users/davidkuna/www/project-base/project-base (Errno::ENOENT)`
|New feature| No <!-- Do not forget to update CHANGELOG.md and possibly docs/ -->
|BC breaks| No <!-- Do not forget to update UPGRADE.md -->
|Fixes issues| ...
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

MattCzerner commented:
Hey David, thank you for your contribution.
Requested changes are correct, but it should be done in `project-base/docker/conf/docker-sync.yml.dist` and not in monorepo one. Because you are changing `docker-sync.yml.dist` in root of `shopsys/shopsys` package, in which relative path to project-base is correct. 

Can you fix it in correct file?

DavidKuna commented:
Hi Mathew, 
I get it now! 😄 

I've moved changes into project-base/docker/conf/docker-sync.yml.dist. 

And what about the `sync_excludes: ['project-base/docker/']`? It seems to me wrong as well so I've changed it.

MattCzerner commented:
Hi David,
apologize for my late response.

Yes, `sync_excludes` is wrong too, and you fixed it right.

I rebased it onto our master, changed commit message, mentioned you in CHANGELOG and merged to master manually in a55c1d50dc830fe732e7e2382e0e207b03de3520.

Thank you for your contribution :)
